### PR TITLE
Feat/dep creation page/shital

### PR DESCRIPTION
**Pull request by sid4848**
This is pull request for depCreationPage.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +17,7 @@
     "@pinia/nuxt": "^0.4.2",
     "@tailwindcss/forms": "^0.5.3",
     "@types/chart.js": "^2.9.37",
+    "@vue/cli-service": "^5.0.8",
```
Comment (panthsolanki) [inline package.json]: why is @vue/cli-service needed

File: pages/careers/depCreationPage.vue [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        <div class="lg:col-start-2 lg:col-span-10">
+          <div
+            class="rounded-md shadow dark:shadow-gray-800 bg-white dark:bg-slate-900 p-6"
+          >
+            <form @submit.prevent="submitForm">
+              <div class="grid lg:grid-cols-12 gap-6">
+                <div class="lg:col-span-6">
+                  <div class="ltr:text-left rtl:text-right">
+                    <label for="name" class="font-semibold">Name:</label>
+                    <input
```
Comment (panthsolanki) [inline pages/careers/depCreationPage.vue]: We have to use existing component instead of input

**Review by panthsolanki — COMMENTED** [review]
Please remove unwanted file

**Review by sid4848 — COMMENTED** [review]
empty file for depcreationpage is created

File: package.json (hunk 1 above)
Comment (sid4848) [reply package.json]: when I tried to run it showed an error. to resolve the error  I had to install this package

File: pages/careers/depCreationPage.vue (hunk 2 above)
Comment (sid4848) [reply pages/careers/depCreationPage.vue]: This is what is available in a reusable components file. you can check.

**Review by panthsolanki — CHANGES_REQUESTED** [review]
please remove unwanted files changes and unwanted packages

**Pull request closed without merging**